justfile

```diff
@@ -0,0 +1,58 @@
+alias e := export
+alias ea := export-all
+alias r := refresh
+alias ra := refresh-all
+alias u := update
+alias ua := update-all
+
+set windows-shell := ["powershell.exe", "-NoLogo", "-Command"]
+
+default:
+    @just --list
+
+[private]
+[windows]
+_batchcmd cmd loader:
+    Get-ChildItem -Path versions\{{ loader }} -Directory | % { \
+      Set-Location $_.FullName; \
+      Write-Host "running {{ cmd }} in $_.FullName"; \
+      Invoke-Expression "{{ cmd }}"; \
+      Pop-Location; \
+    }
+
+[linux]
+[macos]
+[private]
+_batchcmd cmd loader:
+    for d in versions/{{ loader }}/*/; do \
+      cd "$d"; \
+      echo "running {{ cmd }} in $d..."; \
+      {{ cmd }}; \
+      cd {{ invocation_directory() }}; \
+    done
+
+# all versions of <loader> will be exported as a modrinth modpack
+[linux]
+[macos]
+export loader: && (_batchcmd "packwiz modrinth export" loader)
+    -mkdir -p .build/{{ loader }}
```

Comment:
This exports it to `versions/modloader/mcversion` instead of `.build/loader` btw.

Reply:
fixed. everything will export to `build/` now, which i think makes more sense as the difference between loader doesn't matter much anyways when exporting; plus you could export the packs for a specific loader if need be